### PR TITLE
Added Studio 3T

### PR DESCRIPTION
**Pull request by aalfath**
NOTE: A similar PR may already be submitted! Please search among the Pull request before creating one.

### Types of Changes

**What types of changes does your PR introduce?** Put an x in all boxes that apply

- [x] New addition to list
- [ ] Fixing bug in existing item in list
- [ ] Removing item from list
- [ ] Changing structure (organization) of list

### Proposed Changes

**Describe each of your changes here to communicate to the maintainers why we should accept this pull request.**

I added Studio 3T. I think it deserves to be in the list since it is one of the most used MongoDB GUI client on Mac. It is an official partner of MongoDB as you can check out here: https://www.mongodb.com/partners/studio-3t

### PR Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read the CONTRIBUTING guide
- [x] I have explained why this PR is important
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -507,6 +507,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 *  [Sequel Ace](https://github.com/Sequel-Ace/Sequel-Ace) - The maintained "sequel" to the longtime macOS tool Sequel Pro. [![Open-Source Software][oss icon]](https://github.com/Sequel-Ace/Sequel-Ace)
 * [SQLight](https://aurvan.com/sqlight/) - SQLite database manager tool. ![Freeware][Freeware Icon]
 * [SQLPro Studio](http://www.sqlprostudio.com/) - Simple, powerful database manager for macOS.
+* [Studio 3T](https://studio3t.com/) - the Ultimate GUI for MongoDB. ![Freeware][Freeware Icon]
```
Comment (jaywcjlove) [inline README.md:510]: It's not free.

@aalfath

Comment (aalfath) [reply README.md:510]: Hi @jaywcjlove, it actually has a free edition as described here: https://studio3t.com/free/

Shall I change the URL to that specific link instead?

Thanks for checking out this PR!

Comment (jaywcjlove) [reply README.md:510]: @aalfath Is this its purchase interface? https://studio3t.com/buy/

Comment (aalfath) [reply README.md:510]: @jaywcjlove that's the purchase page for the non-free edition. the free one does not need to be purchased.

**Pull request merged**